feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.spec.ts

```diff
@@ -379,7 +448,62 @@ describe('ConfigProductTitleComponent', () => {
       );
     });
 
-    it('should not emit in case an order bound configuration does not have the OV aspect (yet)', () => {
+    it('should get product name as part of product configuration, in case configuration is saved cart bound via config product code', () => {
+      setDataForSavedCartEntry();
+      mockConfiguration.productCode = PRODUCT_CODE;
+      mockRouterData.productCode = undefined;
+      initialize();
+
+      expect(productService.get).toHaveBeenCalledWith(
+        PRODUCT_CODE,
+        ProductScope.LIST
+      );
+    });
+
+    it('should get product name as part of product configuration, in case configuration is saved cart bound via routerData product code', () => {
```

Comment:
Could you please also rephrase other test descriptions like proposed above. Please try to build meaningful sentences ('in case configuration is saved cart bound via routerData product code' is hard to understand)